src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -30,7 +31,16 @@ We help you get to the heart of your system by offering two quick and easy ways
         src={nrqlconsole}
     />    
 
-You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages. 
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages.
+
+New Relic AI leverages a variety of skills and tools to complete analysis on your behalf, including:
```

Comment:
I'd reframe this entire list. Instead of things it "provides", talk about what the user can do with it:

.... including:

* Transform natural language into queries (NRQL/Nerdgraph)
* Detect anomalies
* Analyze error logs
* Analyze stack traces
* Summarize New Relic platform concepts from docs
* Summarize and analyze dashboards

Reply:
I'd probably rewrite that list though, that was a quick idea